src/components/Navigator.js

```diff
@@ -393,6 +399,7 @@ class Navigator extends React.Component {
       });
       return;
     }
+    store.dispatch(setCurrentOpenFileNameAction({sourceName: newName, viewName: ''}));
```

Comment:
Do we need a similar line for renaming view files or is that covered somewhere else?